core/src/main/java/org/eclipse/hono/config/ProtocolAdapterProperties.java

```diff
@@ -130,4 +131,27 @@ public boolean isDefaultsEnabled() {
     public void setDefaultsEnabled(final boolean flag) {
         this.defaultsEnabled = flag;
     }
+
+    /**
+     * Gets the timeout value used by protocol adapter liveness check,
+     * which determines if protocol adapter vert.x event loop is blocked.
+     * <p>
+     * Default value of the timeout is 5000 milliseconds.
+     *
+     * @return The timeout value in milliseconds.
+     */
+    public final long getEventLoopBlockedCheckTimeout() {
```

Comment:
IMHO the final must be eliminated here. Or is there any reason to prevent specifically this method to be overwritten by childs?

Reply:
No specific reason. Removed 'final'.